scripts/update_rif_table.py

```diff
@@ -0,0 +1,133 @@
+            for chunk in resp.iter_content(chunk_size=8192):
+                downloaded_file.write(chunk)
+
+
+def read_tsv_file(fname: pathlib.Path) -> Generator:
+    """Load tsv file from NCBI"""
+    with gzip.open(fname, mode="rt") as gz_file:
+        reader = csv.DictReader(gz_file, delimiter="\t", quoting=csv.QUOTE_NONE)
+        yield from reader
+
+
+def parse_gene_info_from_ncbi(fname: pathlib.Path) -> Dict[str, str]:
+    """Parse gene_info into geneid: symbol pairs"""
+    genedict: Dict[str, str] = {}
+    for row in read_tsv_file(fname):
+        if row["#tax_id"] not in TAX_IDS:
+            continue
+        gene_id, symbol = row["GeneID"], row["Symbol"]
+        genedict[gene_id] = symbol
+    return genedict
+
+
+def update_rif(sqluri: str):
+    """Update GeneRIF_BASIC table"""
+    with TemporaryDirectory() as _tmpdir:
+        tmpdir = pathlib.Path(_tmpdir)
+        gene_info_path = tmpdir / "gene_info.gz"
+        download_file(GENE_INFO_URL, gene_info_path)
+
+        generif_basics_path = tmpdir / "generif_basics.gz"
+        download_file(
+            GENERIFS_BASIC_URL,
+            generif_basics_path,
+        )
+        genedict = parse_gene_info_from_ncbi(gene_info_path)
+        insert_query = """
+                                INSERT IGNORE INTO GeneRIF_BASIC
+                                SET GeneRIF_BASIC.`SpeciesId`=%s,
+                                        GeneRIF_BASIC.`GeneId`=%s,
+                                        GeneRIF_BASIC.`symbol`=%s,
+                                        GeneRIF_BASIC.`PubMed_ID`=%s,
+                                        GeneRIF_BASIC.`createtime`=%s,
+                                        GeneRIF_BASIC.`comment`=%s,
+                                        GeneRIF_BASIC.`TaxID`=%s,
+                                        VersionId=%s
+                                """
+
+        with database_connection(sql_uri=sqluri) as con:
+            with con.cursor() as cursor:
+                for row in read_tsv_file(generif_basics_path):
+                    if row["#Tax ID"] not in TAX_IDS:
+                        continue
+                    species_id = TAX_IDS[row["#Tax ID"]]
+                    symbol = genedict.get(row["Gene ID"], "")
+                    insert_values = (
+                        species_id,  # SpeciesId
+                        row["Gene ID"],  # GeneId
+                        symbol,  # symbol
+                        row["PubMed ID (PMID) list"],  # PubMed_ID
+                        row["last update timestamp"],  # createtime
+                        row["GeneRIF text"],  # comment
+                        row["#Tax ID"],  # TaxID
+                        VERSION_ID,  # VersionId
+                    )
+                    cursor.execute(insert_query, insert_values)
+        print(
+            f"Generif_BASIC table updated. In case of error, you can do use VersionID={VERSION_ID} to find rows inserted with this script"
```

Comment:
*nitpick:* fix the grammar in this message.
```suggestion
            f"Generif_BASIC table updated. In case of error, you can use VersionID={VERSION_ID} to find rows inserted with this script"
```

Reply:
Done